src/search/pdbs/utils.cc

```diff
@@ -5,6 +5,7 @@
 #include "pattern_information.h"
 
 #include "../utils/logging.h"
+#include "../utils/rng.h"
 
 #include "../task_proxy.h"
 
```

Comment:
For some reason, I cannot comment in the right line here, this comment belongs to line 14.

I'd use an overflow safe multiplication here, e.g. by adding an `is_product_within_limit` check (as defined in `src/search/utlis/math.h`).

Reply:
It is a bit redundant to add this because the assumption is that you only want to compute the PDB size of a pattern if you also have a PDB for it. The PDB itself, however, can only be constructed if there is no overflow when computing the hash multipliers. I added the suggested safe-guard anyway.